tests/test_cli/test_ipfs_hash.py

```diff
@@ -42,7 +42,8 @@ def test_invocation(
 
         tree = DependencyTree.generate(self.packages_dir_path)
         packages: Set[PackageId] = set()
-        _ = [packages.update(level) for level in tree]
+        for level in tree:
+            packages.update(level)
```

Comment:
there is also a `consume` function now: `consume(map(packages.update, tree))`

https://github.com/valory-xyz/open-aea/blob/068e5861c117f70d6db81e55cfe6c9f845abc3d0/aea/test_tools/utils.py#L49-L51